handlers/_swagger.js

```diff
@@ -210,7 +211,7 @@ function handleMulterConfig(multerConfig, logger, serviceLoader) {
  * handlerFunc - handler callback function
  * logger - the logger service
  */
-function registerRoute(app, auth, additionalMiddleware, method, path, data, allowedTypes, handlerFunc, logger) {
+function registerRoute(app, auth, additionalMiddleware, method, path, data, allowedTypes, handlerFunc, logger, swaggerDoc) {
```

Comment:
given the usage below for `validateRequestParameters` (and elsewhere in this project), it's probably better to have `logger` be the last parameter to the function